tests/task_t_1.spec.js

```diff
@@ -10,4 +10,41 @@ test('duck duck go T 1', async ({ page }) => {
   await page.goto('https://duckduckgo.com/');
   const logo = page.locator('#logo_homepage_link');
   await expect(logo).toBeVisible;
+});
+
+test('T2: Check that search results contain expected text', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('Test');
+  await page.click('#search_button_homepage');
+  const searchResults = page.locator('#links').toBeVisible;
+  const firstResult = await page.textContent('#r1-0');
+  await expect(firstResult).toContain('Test');
+});
+
+test('T3: Check that Cheat sheet for MSWord is displayed', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('Microsoft word cheat sheet');
+  await page.click('#search_button_homepage');
+  const firstResult = await page.textContent('.c-base__title');
+  await expect(firstResult).toContain('Microsoft Word 2010');
+  await page.isVisible('span.chomp--link__mr');
+  expect(await page.click('span.chomp--link__mr')).click;
```

Comment:
I see two clicks in this line, last click does not seem to be needed

Reply:
fixed